services/README.md

```diff
@@ -62,26 +62,29 @@ systemctl unset-environment SCX_FLAGS_OVERRIDE
 systemctl restart scx
 ```
 
-## Checking Journald Logs
+## Checking journald Logs
 
-The SCX schedulers do not log to the default journald namespace. Instead, they save logs in a dedicated ```sched-ext``` namespace.
-This is where you should look for information about possible errors.
 
 - To view the logs, use the following command:
 
 ```
-journalctl --namespace=sched-ext
+journalctl -u scx.service
 ```
 
-- To find logs from another system startup and identify when a potential error might have occurred, use:
+- To view the logs of the current session use the command
 
 ```
-journalctl --namespace=sched-ext --list-boots
+journalctl -u scx.service -b 0
 ```
 
-- To verify the amount of space taken up by the logs, use:
+## Checking scx_stats
+
+- To check the scheduler statistics, use the
 
 ```
-journalctl --namespace=sched-ext --disk-usage
+scx_SCHEDNAME --monitor X
 ```
 
+X - for example 1 - this will print the output every second
+
+
```

Comment:
And maybe add a real example, like:
```
scx_bpfland --monitor 0.5
```

To make it even more clear and to show that it also supports numbers with decimals.

Reply:
I agree adding real examples would be nicer here.
For example, in case of LAVD, its monitoring option is a bit different from the others. 

    /// Run in monitoring mode. Show the specified number of scheduling
    /// samples every second.
    #[clap(long)]
    monitor_sched_samples: Option<u64>,